docs/contents.md

```diff
@@ -1,18 +1,44 @@
 # Qooxdoo Manual (v${qooxdoo.version})
 
-- [Qooxdoo Core](core/): a dedicated class system with custom class properties,
-data binding and a runtime environment
-- [Qooxdoo UI](gui/)
-- [Layouts](layout/)
-- [Qooxdoo Widgets](widget/)
-- [Testing](development/testing/)
-- [Communication](communication/)
-- Deployment (tbd)
-- Qooxdoo Tools
-  - ["qx" CLI commands](cli/commands.md)
-  - [Package system](cli/packages.md)
-  - [Compiler Configuration](configuration/overview.md)
-  - [Compiler API](development/compiler/API.md)
-- [Qooxdoo for Node](server/)
-- [Qooxdoo for Mobile](mobile/)
-- [Tutorials](tutorial/)
+This is an overview of Qooxdoo's comprehensive feature-set and the extensive
+documentation that comes with it. The [sidebar](_sidebar.md) contains a more
+detailed table of contents.
+
+## [Qooxdoo Core](core/) 
+
+A dedicated class system with custom class properties, data binding and a 
+runtime environment.
+
+## [Qooxdoo UI](gui/)
+
+The UI layer of Qooxdoo, providing a rich set of widget which can be used to
+create professional web-based applications.
+
+## [Communication](communication/)
+
+Qooxdoo provides tree different communication APIs with a rich feature set: 
+HTTP Request APIs, a REST Interface which support databinding, and a JSONRPC 
+Client.
+
+## [Qooxdoo on the Server](server/)
+
+If you like to use the same development paradigm for the client and the server,
+Qooxdoo has you covered - Use Qooxdoo's class system and tooling with NodeJS or 
+Rhino.
```

Comment:
Do we still need to mention Rhino? Hasn't it been pretty obsolete, with no maintenance, for many years?

Reply:
It is still in the documentation. But I can remove it since probably no-one uses it anymore.